### PR TITLE
UI: Remove unused & duplicated code

### PR DESCRIPTION
**Pull request by mbelsky**
# What I did

Hey,

I've found that `/lib/ui/src/libs` files are unused or have some duplication.

## How to test

- Check `lib/ui/src/settings/shortcuts.test.js` is passed

### CODE REVIEW

File: lib/ui/src/settings/shortcuts.tsx [hunk 1]
```diff
@@ -200,7 +203,10 @@ class ShortcutsScreen extends Component<ShortcutsScreenProps, ShortcutsScreenSta
 
     this.setState({
       activeFeature: focusedInput,
-      shortcutKeys: { ...shortcutKeys, [focusedInput]: { shortcut: null, error: false } },
+      shortcutKeys: {
```
Comment (mbelsky) [inline lib/ui/src/settings/shortcuts.tsx:206]: Auto formatted on pre-commit hook

File: lib/ui/src/containers/menu.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React, { useMemo } from 'react';
 import { Badge } from '@storybook/components';
 import { API } from '@storybook/api';
 
-import { shortcutToHumanString } from '../libs/shortcut';
+import { shortcutToHumanString } from '@storybook/api/dist/lib/shortcut';
```
Comment (shilman) [inline lib/ui/src/containers/menu.tsx:6]: we shouldn't be importing from `dist`. can you export these from `@storybook/api/shortcut` maybe?

**Review by shilman — CHANGES_REQUESTED** [review]
Good catch 🤦  ... amazing what sneaks into a large codebase with 1000 contributors. Small comment, otherwise looks great.

File: lib/ui/src/containers/menu.tsx (hunk 2 above)
Comment (mbelsky) [reply lib/ui/src/containers/menu.tsx:6]: To be able use the `@storybook/api/shortcut` import I also need add the `shortcut` module declaration in `lib/api/src/typings.d.ts`. And `@storybook/api/shortcut` won't be typed and won't make auto-suggestions while coding.

As an alternative I can export `shortcut` from `lib/api/src/index.tsx`:

```javascript
import * as shortcut from './lib/shortcut';

export { shortcut };
```

And use it like

```javascript
import { shortcut } from '@storybook/api';
const { shortcutToHumanString } = shortcut;
```

Which way is preferred to make `shortcut` importable?

Comment (shilman) [reply lib/ui/src/containers/menu.tsx:6]: You can also add typings for the `shortcut` entry. Here's an example of that:

https://github.com/storybookjs/storybook/blob/next/lib/components/html.js
https://github.com/storybookjs/storybook/blob/next/lib/components/html.d.ts

Comment (mbelsky) [reply lib/ui/src/containers/menu.tsx]: Thank you this example, fixed

**Review by shilman — APPROVED** [review]
Awesome. Thanks for the fix!!!

**Pull request merged**